CHANGELOG.md

```diff
@@ -838,6 +838,11 @@
 * Catch more valid `no_magic_numbers` violations.  
   [JP Simard](https://github.com/jpsim)
 
+* Rewrite `SwiftLintPlugin` using `BUILD_WORKSPACE_DIRECTORY` without relying
+  on the `--config` option.  
+  [Garric Nahapetian](https://github.com/garricn)
+  [#4758](https://github.com/realm/SwiftLint/pull/4758)
```

Comment:
Omit this line when there's no associated issue:
```suggestion
```
The PR number will be attached to the commit message anyway.